src/waitress/task.py

```diff
@@ -385,7 +381,7 @@ def start_response(status, headers, exc_info=None):
                         # 1. "service" method in task.py
                         # 2. "service" method in channel.py
                         # 3. "handler_thread" method in task.py
-                        reraise(exc_info[0], exc_info[1], exc_info[2])
+                        raise exc_info[1]
```

Comment:
not sure this is the same

Reply:
Feel free to peruse this code:

https://github.com/Pylons/waitress/blob/master/src/waitress/compat.py#L87

Since the traceback is grabbed from `exc_info[2]` (which is `exc_info[1].__traceback__`) and `exc_info[0]` is the `tp` value, it is ignored since `value` (which is `exc_info[1]`) ... so ultimately it ends up calling `raise value` which is the same as `raise exc_info[1]`.